public/js/custom.js

```diff
@@ -0,0 +1 @@
+// This file can be edited via the Customization Settings on the Settings Page of the Dashboard!
```

Comment:
the same as the image names, if you use ''custom.js" for all javascript, users need to bust all caches and on the app and also on cloudflare, it might be better to have a script tag in the head, and load the js into that.